### PR TITLE
Add policy docs in the community handbook

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Congrats Liz and Anne for merging the Accessibility Policy draft in the GitHub

With this PR I am creating a linked document to display the contributing guideline and accessibility policy documents in the book (without making a duplicate draft)

### List of changes proposed in this PR (pull-request)

* Add accessibility policy doc
* Add contributing policy doc

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Review by aleesteele — APPROVED** [review]
Thank you for these edits! 🫶

**Comment by aleesteele** [comment]
_Adding breadcrumbs to #3581_

**Comment by aleesteele** [comment]
_Adding breadcrumbs to #3581_

File: book/website/community-handbook/accessibility.md [hunk 1]
```diff
@@ -3,6 +3,14 @@
...
+We aim to draw from the [access-centered framework](https://accesscenteredmovement.com/what-access-centered-means/) in describing access as a constant process that changes in each space with each individual.
+Much like how _The Turing Way_ itself is always being written and rewritten, our accessibility and access-related practices are constantly being updated as broader awareness grows and changes in technology are made.
+
+This chapter contains following subchapters with accessibility guidelines when participating _The Turing Way_ community and recommendations when writing chapters:
+- {ref}`Accessibility Policy of The Turing Way Community<accessibility-policy>`
+- {ref}`accessibility-event-organising`
+- {ref}`accessibility-comms-platforms`
+- {ref}`accessibility-alt-text`
```
Comment (malvikasharan) [inline book/website/community-handbook/accessibility.md:16]:
```suggestion
- {ref}`Accessibility Policy of The Turing Way Community<ch-accessibility-policy>`
- {ref}`ch-accessibility-events`
- {ref}`ch-accessibility-comms-platforms`
- {ref}`ch-accessibility-alttext`
```

File: book/website/community-handbook/contributing-guideline.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+(contributing-guideline)=
+
+```{note}
+This page displays the [`CONTRIBUTING.md`](https://github.com/the-turing-way/the-turing-way/blob/main/CONTRIBUTING.md), a base file with guidelines for contributing to _The Turing Way_.
+```
+
+```{include} ../../../CONTRIBUTING.md
```
Comment (bsipocz) [inline book/website/community-handbook/contributing-guideline.md:7]: This caused #3976 

I suppose we could hardwire the absolute URL for CoC

Comment (malvikasharan) [reply book/website/community-handbook/contributing-guideline.md:7]: I was wondering how many link on this page might be broken when navigating from the book. I can edit the contributing file to add full links. Sorry about this.

Comment (bsipocz) [reply book/website/community-handbook/contributing-guideline.md:7]: It is all fixed now, I only commented at the end of the detective mission to see if something is broken in our CI or not.

 But it all worked as expected, it was just an oversight that the PR got merged with build issues.

**Pull request merged**